modules/@apostrophecms/doc-type/index.js

```diff
@@ -2682,8 +2682,11 @@ module.exports = {
               subquery.limit(undefined);
               subquery.page(undefined);
               subquery.perPage(undefined);
-              const mongo = await subquery.toMongo();
-              const count = await mongo.count();
```

Comment:
We should try to find a way to make `count()` work on cursors for better bc. Can you extend cursor objects the same way you extend the others? The methods of a collection that return a cursor could be wrapped to attach a collection property to that cursor so that your added `count()` method can work, unless I am missing something. Possibly the criteria used to create the cursor are hard to access when extending cursor?

Reply:
I've restored the `count` method. It was a journey to implement 😄